README.md

```diff
@@ -249,6 +249,23 @@ Kubernetes secrets:
 $ kubectl create secret generic spid-cert --from-file=./cert.pem --from-file=./key.pem
 ```
 
+### SAML (SPID) certs with ENV vars
+
+The certificate and the secret key could be provided to the backend with two ENV variables:
+- `SAML_KEY`
+- `SAML_CERT`
+
+if these variables are set `cert.pem` and `key.pem` are ignored.
+
+### SPID Identity Providers Metadata
+
+When backend starts, SPID login is configured with Identity Providers metadata fetched from remote URL. URLs are provided by the following Environment variables: `CIE_METADATA_URL` (CIE login) `IDP_METADATA_URL` (SPID login) and `SPID_TESTENV_URL` (SPID test environment). Before that the backend become ready a time consuming operation must be completed (fetching remote metadata).
```

Comment:
can you add a table with the default values from both test (local development) and prod environments ?

Reply:
Added into 344c47b